saleor/graphql/schema.graphql

```diff
@@ -781,6 +781,7 @@ type Channel implements Node {
   slug: String!
   currencyCode: String!
   hasOrders: Boolean!
+  shippingZones: [ShippingZone]
```

Comment:
Does it return `null` or empty list when there are no shipping zones? I'm wondering if we can change it to: `shippingZones: [ShippingZone!]!`.

Reply:
The empty list is returning. I can change it to `shippingZones: [ShippingZone!]!`.